src/SIL.LCModel.Core/SpellChecking/SpellEngine.cs

```diff
@@ -75,9 +74,10 @@ public bool Check(string word)
 		/// </summary>
 		/// <param name="word"></param>
 		/// <returns></returns>
-		private static byte[] MarshallAsUtf8Bytes(string word)
+		private static string MarshallAsUtf8Bytes(string word)
 		{
-			return Encoding.UTF8.GetBytes(Icu.Normalize(word, Icu.UNormalizationMode.UNORM_NFC) + "\0");
+			byte[] bytes = Encoding.UTF8.GetBytes(Icu.Normalize(word, Icu.UNormalizationMode.UNORM_NFC) + "\0");
```

Comment:
I'm not sure that this method is necessary anymore. The Icu.Normalize call may be all that is necessary since we don't need to pass a byte array to NHunspell.